--- modxslt/mod_xslt.c.orig	Fri Sep 13 17:24:24 2002
+++ modxslt/mod_xslt.c	Fri Sep 13 17:24:05 2002
@@ -57,7 +57,7 @@
     int debug;
 } mod_xslt_cfg;
 
-module MODULE_VAR_EXPORT mod_xslt_module;
+module MODULE_VAR_EXPORT xslt_module;
 
 const char *mod_xslt_dyndoc_cmd(cmd_parms *cmd, void *m, int flag);
 const char *mod_xslt_dynss_cmd(cmd_parms *cmd, void *m, int flag);
@@ -205,7 +205,7 @@
 
     void *theproc;
     mod_xslt_cfg *cfg = (mod_xslt_cfg*) 
-	ap_get_module_config(r->per_dir_config, &mod_xslt_module);
+	ap_get_module_config(r->per_dir_config, &xslt_module);
     se = SablotCreateProcessor (&theproc);
     if (cfg->debug)
       se |= SablotRegHandler(theproc, HLR_MESSAGE, &sablotMH, (void*)r);
@@ -246,7 +246,7 @@
   int cnt1;
   request_rec * subr = NULL;
   mod_xslt_cfg *cfg = (mod_xslt_cfg*) 
-	ap_get_module_config(r->per_dir_config, &mod_xslt_module);
+	ap_get_module_config(r->per_dir_config, &xslt_module);
 
 
   /* Setup some common strings of text, for file not found, etc. */
@@ -455,7 +455,7 @@
 };
 
 /* Dispatch list for API hooks */
-module MODULE_VAR_EXPORT mod_xslt_module = {
+module MODULE_VAR_EXPORT xslt_module = {
     STANDARD_MODULE_STUFF, 
     mod_xslt_init,                  /* module initializer                  */
     mod_xslt_create_dir_cfg,    /* create per-dir    config structures */
